docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -64,7 +64,7 @@ Set the systemd unit name separator between the name/id of a container/pod and t
 Generate a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout to stdout. Note that the **RequiresMountsFor** option in the **Unit** section ensures that the container storage for both the GraphRoot and the RunRoot are mounted prior to starting the service. For systems with container storage on disks like iSCSI or other remote block protocols, this ensures that Podman is not executed prior to any necessary storage operations coming online.
 
 ```
-$ podman create --name nginx nginx:latest
+$ podman create --name nginx:latest
```

Comment:
--name does take a value.

This is creating a container named nginx off of the nginx:latest image.